### ISSUE TITLE
Spaces before colons in selectors are meaningful

### ISSUE THREAD

**Issue by bleargh45** [bleargh45/CSS-Minifier-XS#1]
Migrated from [rt.cpan.org #97574](https://rt.cpan.org/Ticket/Display.html?id=97574) (status was 'new')

Requestors:
* dnebdal@gmail.com

From https://eu.wargaming.net/id/511501418-dnebdal/ on 2014-07-29 00:05:44
:

Given this CSS: 
#test :link, #test :visited {color: green}

The current minified CSS is:
#test:link,#test:visited{color:green} 

That doesn't mean exactly the same, though - the original is "elements that are links within an element with id 'test'", while the minified only applies to an element if it has both the id 'test' and is a link. 

In other words, whitespace before a colon in selectors is, sadly, meaningful (but can be collapsed to a single space).

**Comment by bleargh45** [comment]
Ok... this should now be fixed by `6141a22`.  Release pending.
